manifests/03-metrics-service.yaml

```diff
@@ -12,8 +12,9 @@ spec:
   clusterIP: None
   ports:
   - name: https
-    port: 9192
-    targetPort: https
+    protocol: TCP
```

Comment:
why?

Reply:
metrics are already being reported https://xlugntax-promecieus.svc.ci.openshift.org/graph?g0.range_input=1w&g0.stacked=1&g0.moment_input=2019-10-15%2014%3A55%3A54&g0.expr=mapi_max_pending_csr&g0.tab=0

Reply:
Thanks, confused the named port.